Docs: Contributing folder (#4415)
langflow-ai/langflow: docs/docs/Contributing/contributing-telemetry.md

slug: /contributing-telemetry
---

> Langflow uses anonymous telemetry to collect essential usage statistics to enhance functionality and user experience. This data helps us identify popular features and areas needing improvement, and ensures development efforts align with what you need.

Reviewer: ```suggestion
Langflow uses anonymous telemetry to collect essential usage statistics to enhance functionality and the user experience. This data helps us identify popular features and areas that need improvement, and ensures development efforts align with what you need.
```

We respect your privacy and are committed to protecting your data. We do not collect any personal information or sensitive data. All telemetry data is anonymized and used solely for improving Langflow.

> You can opt-out of telemetry by setting the `LANGFLOW_DO_NOT_TRACK` or `DO_NOT_TRACK` environment variable to `true` before running Langflow. This will disable telemetry data collection.

Reviewer: I think this information gets lost. It should be under a heading "Opt out of telemetry". Also, remove the hypen from "opt out" in the sentence.

> You can opt out of telemetry by setting the `LANGFLOW_DO_NOT_TRACK` or `DO_NOT_TRACK` environment variable to `true` before running Langflow. This will disable telemetry data collection.

Reviewer: ```suggestion
To opt out of telemetry, set the `LANGFLOW_DO_NOT_TRACK` or `DO_NOT_TRACK` environment variable to `true` before running Langflow. This disables telemetry data collection.
```

> This telemetry data is crucial for enhancing Langflow and ensuring that our development efforts align with your needs. Your feedback and suggestions are invaluable in shaping the future of Langflow, and we appreciate your support in making Langflow better for everyone.

Reviewer: You can remove this sentence. It's basically a repeat of a previous sentence.

> ## Telemetry Data Collected

Reviewer: ```suggestion
## Data that Langflow collects
```
